Requirements/Requirements_And_Goals.md

```diff
@@ -0,0 +1,73 @@
+# Requirements and Goals
+
+## Mission
+
+> The Hey, Blue! initiative facilitates moments of meaningful connection between police officers and members of their community by offering intentional opportunities for individuals to meet and share positive experience.
+
+## Goal
+
+To provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. This platform can be hosted on both web and mobile based. 
```

Comment:
`helps civilians` not `helps the civilians`
`at scale` not `at a scale`
`This platform must be accessible on both mobile and web`